modules/ztp-adding-new-content-to-gitops-ztp.adoc

```diff
@@ -26,17 +26,17 @@ example
     └── source-crs <1>
         ├── PaoCatalogSource.yaml
```

Comment:
```suggestion
        ├── PaoCatalogSource.yaml <1>
```
I think it would be good to highlight that this file is in fact a custom version of the file provided in the `ztp-site-generate` container.

Reply:
ie, you can provide wholly new files, and you can provide customized versions of existing files using this custom CRs method.